eng/code-quality-reports/src/main/resources/spotbugs/spotbugs-include.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<FindBugsFilter xmlns="https://github.com/spotbugs/filter/3.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                xsi:schemaLocation="https://github.com/spotbugs/filter/3.0.0 https://raw.githubusercontent.com/spotbugs/spotbugs/3.1.0/spotbugs/etc/findbugsfilter.xsd">
+  <Match>
+    <Not>
+      <Bug pattern="EI_EXPOSE_REP,EI_EXPOSE_REP2"/>
```

Comment:
This is pretty cool to configure this globally and not have to add these suppressions in every library!